shell/platform/embedder/embedder.cc

```diff
@@ -194,24 +198,29 @@ InferOpenGLPlatformViewCreationCallback(
       gl_proc_resolver,                    // gl_proc_resolver
   };
 
-  return [gl_dispatch_table, fbo_reset_after_present,
-          platform_dispatch_table](flutter::Shell& shell) {
-    return std::make_unique<flutter::PlatformViewEmbedder>(
-        shell,                    // delegate
-        shell.GetTaskRunners(),   // task runners
-        gl_dispatch_table,        // embedder GL dispatch table
-        fbo_reset_after_present,  // fbo reset after present
-        platform_dispatch_table   // embedder platform dispatch table
-    );
-  };
+  return fml::MakeCopyable(
+      [gl_dispatch_table, fbo_reset_after_present, platform_dispatch_table,
+       external_view_embedder =
```

Comment:
There is no variable shadowing in this case because there is no implicit capture of the external view embedder. Also, I think `-Wshadow` which is on by default would have caught this form of shadowing.

Reply:
I was wrong https://github.com/flutter/engine/pull/10480